GLib-2.0.gir

```diff
@@ -13802,12 +13834,16 @@ type representing a set of sources to be handled in a main loop.</doc>
           <type name="MainContext" c:type="GMainContext*"/>
         </return-value>
       </constructor>
-      <constructor name="new_with_flags" c:identifier="g_main_context_new_with_flags">
+      <constructor name="new_with_flags" c:identifier="g_main_context_new_with_flags" version="2.72">
```

Comment:
Can remove the configuration for this one and possibly others

Reply:
a bunch of others yes